### DISCUSSION TITLE
大佬，这个应该怎么解决，还有就是不知道为什么，之前安装完补丁之后，时不时就失效了，还要重新安装

### DISCUSSION THREAD

**Discussion by cwjgiegie** [huiyadanli/RevokeMsgPatcher#665] [General]
![图片](https://github.com/huiyadanli/RevokeMsgPatcher/assets/93698092/4f37eaa2-f162-41f3-a9aa-06cf894b6d45)

**Comment by huiyadanli** [comment]
使用最新的1.7版本
